File: 1598_crawler_log_folder.py
from typing import List

def minOperations(logs: List[str]) -> int:
    stack = []

    for cmd in logs:
        if cmd == "./":
            continue
        elif cmd == "../":
            if stack:
                stack.pop()
        else:
            stack.append(cmd)

    return len(stack)

File: go/1598_crawler_log_folder.go
package main

func minOperations(logs []string) int {
	stack := []string{}

	for i := 0; i < len(logs); i++ {
		op := logs[i]

		if op == "./" {
			continue
		} else if op == "../" {
			if len(stack) > 0 {
				stack = stack[:len(stack)-1]
			}
		} else {
			stack = append(stack, op)
		}
	}

	return len(stack)
}
